frontend/src/index.jsx

```diff
@@ -59,6 +72,11 @@ const router = createHashRouter([
         path: "/local-report",
         Component: DragAndDrop,
       },
+      {
+        path: "/implementations/:langImplementation",
+        Component: ImplementationReportView,
+        loader: async () => fetchAllReportData(),
```

Comment:
This makes a thunk but we have one already no, does `loader: fetchAllReportData` not work in Javascript?

Reply:
Yes, it works. Fixed now!